app/dashboard/templates/shared/nav_internal.html

```diff
@@ -18,7 +18,7 @@
 
         <nav class="navbar navbar-expand-md navbar-dark">
             <a class="navbar-brand pt-0 pb-0" href="/">
-                    <img id="logo" src="/static/v2/images/logo_large.png" data-hover="/static/v2/images/logo_large_hover.gif" width="150" height="50" />
+                    <img id="logo" src="{% if request.path == '/ios' %}/static/v2/images/ios/gitcoin.svg{%else%}/static/v2/images/logo_large.png{% endif %}" data-hover="{% if request.path == '/ios' %}/static/v2/images/ios/gitcoin_hover.gif{%else%}/static/v2/images/logo_large_hover.gif{% endif %}" {% if not request.path == '/ios' %}width="150"{%endif%} height="50" />
```

Comment:
this line is hard to maintain from my perspective... id rather wrap the whole `<img>` block inside of the `{% if request.path == '/ios' %}` block

Reply:
https://github.com/gitcoinco/web/pull/207/commits/72c4d59f1dff1fadb716083a4e5b9a2433fef0bb resolves this.